src/pages/collection/Series.tsx

```diff
@@ -63,6 +63,13 @@ const Series = () => {
 
   const dispatch = useDispatch();
 
+  const languageMapping = { 'x-jat': 'ja', 'x-kot': 'ko', 'x-zht': 'zh-hans' };
```

Comment:
Static fields should be declared outside the component

Reply:
👍